packages/spectral/src/types/IntegrationDefinition.ts

```diff
@@ -217,10 +237,14 @@ export type ConnectionConfigVar<TComponents extends ComponentSelector<any>> =
 
 export type ConfigVar<TComponents extends ComponentSelector<any>> =
   | StandardConfigVar
+  | CodeConfigVar
   | ScheduleConfigVar
   | DataSourceConfigVar<TComponents>
   | ConnectionConfigVar<TComponents>;
 
+export const isCodeConfigVar = (cv: ConfigVar<any>): cv is CodeConfigVar =>
```

Comment:
Where is this going to be used?

Reply:
I was initially using it in the `convertIntegration` layer but that ended up not being strictly necessary. I opted to leave it since it's now a formal config var type and the rest of that union have type guard functions.